tests/README.md

```diff
@@ -67,7 +67,106 @@ EOF
 ```
 
 ### Scheduling recurrent test runs.
-For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template. Reffer to the for a sample cron job and pipelinerun temaple. [example.yaml](pipelineruns%2Feks%2Fexample.yaml)
+For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template. Refer to the for a sample cron job and pipelinerun template below.
```

Comment:
nit: `Refer "to the for" a sample cron job and pipelinerun template below.` ->  `Please refer to below example for sample cron job and pipelinerun template`

Reply:
nit: `For scheduling recurrent test runs a cron job is used to trigger the pipeline from the pipelinerun template` --> `To schedule recurrent test runs, you can leverage cron-job to trigger a pipeline run using trigger-template`

Reply:
done